src/components/bulkConcepts/component/ActionButtons.jsx

```diff
@@ -46,13 +46,13 @@ export class ActionButtons extends Component {
     this.props.addConcept(params, data, name);
   };
 
-  addConceptButton = () => {
+  addConceptButton = (id, url, display_name) => {
     notify.show('Adding...', 'warning', 800);
 
-    this.fetchPreview(this.props.id);
```

Comment:
Is this also about testing the untested areas?

Reply:
Yes, this is part of the first bullet point that handles restructuring of the code in order for it to be tested.